Snafu backtrace compilation error when building with streams feature

No need to add any code as it is reproducible (at least for me), in a project not yet using Octocrab. This happens when I enable the streams feature: 

```
error[E0432]: unresolved import `snafu::GenerateBacktrace`
   --> /home/daniel/.cargo/registry/src/github.com-1ecc6299db9ec823/octocrab-0.15.4/src/api/repos/releases.rs:177:13
    |
177 |         use snafu::GenerateBacktrace;
    |             ^^^^^^^^^^^^^^^^^^^^^^^^ no `GenerateBacktrace` in the root

error[E0599]: no function or associated item named `generate` found for struct `snafu::Backtrace` in the current scope
   --> /home/daniel/.cargo/registry/src/github.com-1ecc6299db9ec823/octocrab-0.15.4/src/api/repos/releases.rs:199:46
    |
199 |                 backtrace: snafu::Backtrace::generate(),
    |                                              ^^^^^^^^ function or associated item not found in `snafu::Backtrace`
    |
    = help: items from traits can only be used if the trait is in scope
help: the following trait is implemented but not in scope; perhaps add a `use` for it:
    |
1   | use snafu::GenerateImplicitData;
    |

Some errors have detailed explanations: E0432, E0599.
For more information about an error, try `rustc --explain E0432`.
```

--- a/src/api/repos/releases.rs
+++ b/src/api/repos/releases.rs
@@ -158,7 +158,7 @@ impl<'octo, 'r> ReleasesHandler<'octo, 'r> {
     /// let mut stream = octocrab::instance()
     ///     .repos("owner", "repo")
     ///     .releases()
-    ///     .stream_asset(42usize)
+    ///     .stream_asset(42u64.into())
     ///     .await?;
     ///
     /// while let Some(chunk) = stream.next().await {
@@ -174,7 +174,7 @@ impl<'octo, 'r> ReleasesHandler<'octo, 'r> {
         asset_id: AssetId,
     ) -> crate::Result<impl futures_core::Stream<Item = crate::Result<bytes::Bytes>>> {
         use futures_util::TryStreamExt;
-        use snafu::GenerateBacktrace;
+        use snafu::GenerateImplicitData;
 
         let url = format!(
             "repos/{owner}/{repo}/assets/{asset_id}",

